build.gradle.kts

```diff
@@ -8,29 +8,30 @@ repositories {
 }
 
 dependencies {
-    implementation("org.junit.jupiter:junit-jupiter-api:5.8.1")
-    implementation("org.junit.jupiter:junit-jupiter-engine:5.8.1")
+    testImplementation(platform("org.junit:junit-bom:5.8.2"))
+    testImplementation("org.junit.jupiter:junit-jupiter")
```

Comment:
It seems a little paradoxical to specify a `platform` constraint and only use a single dependency from it, I think it would be cleaner to implement it like it is in #36

Reply:
This seems to be the recommended way to do it (https://github.com/junit-team/junit5-samples/commit/9a78828012612630951df4eaf7f43c727f10b484 [junit5-jupiter-starter-gradle/build.gradle](https://github.com/junit-team/junit5-samples/commit/9a78828012612630951df4eaf7f43c727f10b484#diff-4ecac6ad19fa7447da4e041cd9d8e2b27819254d4da01588d447ee892d3fb313))

Reply:
The use case of a Maven bom or a Gradle bom is to enforce a set of constraints, as only one dependency is being used here isn't entirely redundant.

Reply:
Are you going to suggest a change to the junit sample build.gradle?

Reply:
It's ready in the accepted answer
![image](https://user-images.githubusercontent.com/65446343/144532051-13ef70ab-9ede-45f4-9563-fecda65fdfd5.png)

Reply:
but that is followed by:

> But I would stick to the recommendation of the JUnit team and their sample projects on GitHub.